eng/pipelines/runtime.yml

```diff
@@ -148,3 +148,16 @@ jobs:
       liveCoreClrBuildConfig: checked
       dependsOnBuildConfiguration: Release
       condition: false
+
+#
+# Installer Build and Test
+#
+- template: /eng/pipelines/installer/installer-matrix.yml
+  parameters:
+    liveCoreClrBuildConfig: checked
+    liveLibrariesBuildConfig: Release
+    skipPlatforms:
```

Comment:
I think we should invert this to instead specify the platforms we want to build for, not the platforms we want to skip. We should do this in a similar manner to the `platform-matrix.yml` file is done.

Reply:
Anyway I think it makes sense to add a comment so that whenever we expand the full matrix for all partitions we don't forget adding any platform.